chaincfg/init_test.go

```diff
@@ -184,6 +184,71 @@ func TestRejectDuplicateVoteIDs(t *testing.T) {
 	}
 }
 
+// TestForcedChoice ensures the validate logic for a forced choice works as
+// intended.
+func TestForcedChoice(t *testing.T) {
+	// mockChoices is a series of choices used in the tests below and is defined
+	// here convenience.
+	mockChoices := []Choice{{
+		Id:          "abstain",
+		Description: "abstain voting for change",
+		Bits:        0x0000,
+		IsAbstain:   true,
+		IsNo:        false,
+	}, {
+		Id:          "no",
+		Description: "keep the existing rules",
+		Bits:        0x0002, // Bit 1
+		IsAbstain:   false,
+		IsNo:        true,
+	}, {
+		Id:          "yes",
+		Description: "change to the new rules",
+		Bits:        0x0004, // Bit 2
+		IsAbstain:   false,
+		IsNo:        false,
+	}}
+
+	tests := []struct {
+		name    string   // test description
+		choices []Choice // choices to use for validation
```

Comment:
Having choices as a param seems unnecessary, its hard-coded so every test uses the same.

Reply:
Fair enough.  I generally prefer to do it that way to make it easier to add mutations later if the need arises, but this would be trivial to add if needed.

Updated to use the  `mockChoices` directly.